Content.Server/Radio/EntitySystems/HeadsetSystem.cs

```diff
@@ -111,4 +138,49 @@ private void OnEmpPulse(EntityUid uid, HeadsetComponent component, ref EmpPulseE
             args.Disabled = true;
         }
     }
+
+    private void OnPrototypeReload(PrototypesReloadedEventArgs args)
+    {
+        CacheChannels();
+    }
+
+    private void OnGetVerbs(EntityUid uid, HeadsetComponent component, ref GetVerbsEvent<Verb> args)
```

Comment:
I could be wrong, but I don't think this needs to be in server? If you could put this in shared, it wont have that weird lag when you click before the verb pops up!